### DISCUSSION TITLE
Feature Request: Iso file name labelling/commenting on SD Card

### DISCUSSION THREAD

**Discussion by WhatIsTheWeather** [ZuluSCSI/ZuluSCSI-firmware#413] [Q&A] [answered]
When using the ZuluSCSI I frequently store multiple .iso images on the SD card, sometimes upwards of 50+ that I rotate through for different jobs. Would there be any possibility of adding file labelling/commenting which would be ignorable by the ZuluSCSI for file organisation sake? Say I have an iso on the sd card named CD1.iso, would it be possible to have an ignorable label in the file name, for example, CD1-Audio2.iso, with the ZuluSCSI ignoring anything after the hyphen or another character to denote the label? 

Thanks

**Comment by aperezbios** [comment, marked as answer]
@WhatIsTheWeather  This functionality is already implemented :) 

Anything after the first space in the filename, excluding the file suffix, is ignored by the firmware. I've moved this to the Q&A discussion section so it's more likely someone stumbles across this.
